administration/src/application/components/forms/VolunteerServiceEntitlementForm.tsx

```diff
@@ -34,8 +34,7 @@ const VolunteerServiceEntitlementForm: Form<State, Options, ValidatedInput, Addi
       />
       <h4>Tätigkeitsnachweis</h4>
       <p>
-        Hängen Sie hier bitte einen eingescannten oder abfotografierten Tätigkeitsnachweis an. Die Datei darf maximal{' '}
-        {FILE_SIZE_LIMIT_MEGA_BYTES} MB groß sein und muss im JPG, PNG oder PDF Format sein.
+        Hängen Sie hier bitte einen eingescannten oder abfotografierten Tätigkeitsnachweis an. {FileRequirementsText}
```

Comment:
Shouldn't it be `Falls vorhanden: hängen Sie`_

Reply:
Here, we don't have the user specify an organization, so we should probably keep the certificate upload required. (see comment above/below)